web/client/utils/CoordinatesUtils.js

```diff
@@ -791,6 +791,16 @@ const CoordinatesUtils = {
             zoom: map.zoom,
             crs: 'EPSG:4326'
         };
+    },
+    /**
+     * choose to round or floor value incase of 0 fractional digits
+     * @return the rounded value or the orignal one
+    */
+    checkRounding: ({ roundingBehaviour = "round", value = 0, maximumFractionDigits = 0 } = {}) => {
+        if (maximumFractionDigits === 0 && !!Math[roundingBehaviour]) {
```

Comment:
Math[roundingBehaviour] is truthy, you don't need the !! operator,  think

Reply:
yes i know, i wanted to make it more clear (to newBies).  going to change it.